### ISSUE TITLE
Build fails on latest stable and nightly (Possible fix inside)

### ISSUE THREAD

**Issue by drozdziak1** [JakubOnderka/ip_network#1]
Hi there, 
Building `ip_network` fails for me on the latest stable and nightly toolchains:
```
   stable-x86_64-unknown-linux-gnu unchanged - rustc 1.23.0 (766bd11c8 2018-01-01)
  nightly-x86_64-unknown-linux-gnu unchanged - rustc 1.25.0-nightly (51b0b3734 2018-01-12)
```

**A thing that solved this** for me was to set `default-features` to `true` for the `extprim` dependency.

**Is the solution above correct? Do you happen to know what the root cause might be?** 

Thank you,
Stanisław

**Comment by JakubOnderka** [comment]
Hi and thanks for report!

I think that bug is in extprim 1.5, so I changed version to 1.4 in recently released ip_network crate version 0.1.2.

Extrprim package issue: https://github.com/kennytm/extprim/issues/14

**Comment by drozdziak1** [comment]
Thanks for getting back to me so quick and filing that other issue. 

It seems that for a weird, unknown reason our build only builds extprim 1.5 even though it is specified as the other version.

**However**, extprim 1.5 by itself builds without any problems when manually pulled from [their github](https://github.com/kennytm/extprim). And `ip_network` builds without problems even with 1.5 when `default-features = true`. Why did you turn it to false in the first place?

**Comment by JakubOnderka** [comment]
I think that problem is, that extprim don't use `use-std` when I switch `default-features` to `false`. And `println` macro is part of part of std library. 

And the reason why `ip_network` crate sets `default-features` to `false` is, that it is not necessary to compile unused libraries like serde or rand.

**Comment by drozdziak1** [comment]
Fair enough, I used the `default-features = true` "hack" (we're using serde somewhere else anyway so it's not an issue) in our own fork as a temporary solution. 

Unfortunately our project has a deadline so I won't be able to pursue what might have changed in the latest Rust compilers, but feel free to ask if there's anything else I could tell you about this.

**Comment by JakubOnderka** [comment]
@drozdziak1 It is fixed in extprim 1.5.1. Thanks again for bug report!
